<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workbench_settings_base</class>
 <widget class="QWidget" name="workbench_settings_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QGIS Animation Workbench Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string># QGIS Animation Workbench Settings</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_thread_pool_size">
       <property name="text">
        <string>Number of concurrent render tasks</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spin_thread_pool_size"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="thread_pool_description">
     <property name="text">
      <string>The maximum number of concurrent threads to allow during rendering. Setting to the same number of CPU cores you have would be a good conservative approach.  If you want to produce your animation faster, you could probably run 100 or more on a decently specced machine.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="debug_mode_checkbox">
     <property name="text">
      <string>Enable developer mode</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="debug_mode_description">
     <property name="text">
      <string>This is intended for developers to attach to the plugin using a remote debugger so that they can step through the code. Do not enable it if you do not have a remote debugger set up as it will block QGIS startup until a debugger is attached to the process. In addition, debug mode will enable a 'show commands' button in the user interface to let you preview the ffmpect generated commands, and will write to '/tmp' rather than an ephemeral directory for intermediate products. Requires restart after changing.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="verbose_mode_checkbox">
     <property name="text">
      <string>Verbose logging mode</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="verbose_mode_description">
     <property name="text">
      <string>Adds verbose log message, useful for diagnostics.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
